sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc

```diff
@@ -285,7 +286,8 @@ by this specification. Otherwise, the implementation must issue a proper diagnos
 The following algorithm to determine the sub-group size may be employed:
 - If any of the actual SPMD arguments is non-uniform, the specification requires
 that the corresponding SIMD argument must have `simd<T, N>` or `simd_mask<T, N>`
-type, and `N` must be equal to the the sub-group size. The implementation may
+type, and `N` must be equal to the the sub-group size (or multiple for `flatten`).
+The implementation may
```

Comment:
I don't see how this can work.  Let's say the SIMD function returns an `unflatten<simd<T, 12>>` (using my suggested syntax from my other comment).  How can the implementation use the `12` to deduce the SIMD width?